cypress/integration/augmintbasic.js

```diff
@@ -1,53 +1,6 @@
-/*
-    TODO:
-        - split into multiple
-*/
-
-describe("Augmint basic e2e", function() {
-    const getLoan = (prodId, disbursedAmount, repaymentAmount, ethAmount) => {
-        cy.get("[data-testid=getLoanMenuLink]").click();
-
-        cy.get(`[data-testid=selectLoanProduct-${prodId}]`).click();
-        cy.get("[data-testid=selectedLoanProductBlock]").should("contain", "Selected: loan product " + (prodId + 1));
-        // NB: only works with integers, see: https://github.com/cypress-io/cypress/issues/1171
-        cy
-            .get("[data-testid=loanTokenAmountInput]")
-            .type(disbursedAmount.toString())
-            .should("have.value", disbursedAmount.toString());
-        cy.get("[data-testid=repaymentAmountInput]").should("have.value", repaymentAmount.toString());
-        cy.get("[data-testid=ethAmountInput]").should("have.value", ethAmount.toString());
-
-        cy.get("[data-testid=submitBtn]").click();
-
-        return cy
-            .get("[data-testid=EthSubmissionSuccessPanel]")
-            .within(() => {
-                cy.contains("You've got a loan");
-                cy.contains("Disbursed: " + disbursedAmount + " A-EUR");
-                cy.contains("To be repaid: " + repaymentAmount + " A-EUR");
-                cy.contains("Collateral in escrow: " + ethAmount + " ETH");
-            })
-            .as("@submissionSuccess");
-    };
-
-    before(function() {
-        cy.ganacheTakeSnapshot();
-        cy.issueTo(100000); // to make tests independent. issue to accounts[0] by default (amount with token decimals)
-        cy.visit("/");
-        cy.get("[data-testid=useAEurButton]").click();
-        cy.get("[data-testid=TryItConnectedPanel]").should("contain", "You are connected");
-    });
-
-    beforeEach(function() {
-        cy.getUserAEurBalance().as("startingAeurBalance");
-    });
-
-    after(function() {
-        cy.ganacheRevertSnapshot();
-    });
-
+describe("Augmint base", function() {
     it("Under the hood", function() {
-        cy.visit("/under-the-hood");
+        cy.visit("/under-the-hood"); // TODO: expose link on UI to avoid need for slow reconneect from visit()
```

Comment:
adding it to the footer maybe?

Reply:
https://trello.com/c/V1ye4GdN/61-place-under-the-hood-link-to-the-footer